sdk/react/src/lib/fragment-components/a-time-picker.tsx

```diff
@@ -63,6 +63,22 @@ export const ATimePickerSettingsUI = ({ selectedComponent, setComponent }: any)
 		});
 	};
 	return <>
+		<Checkbox
+			labelText='Timezone'
+			id='light'
+			checked={selectedComponent.timezone}
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				timezone: checked
+		})} />
```

Comment:
Instead can't we just conditionally render based on timezone length? If items length is 0, we don't render